src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -208,10 +208,10 @@
 		<value>Dodaj mod (Zaawansowane)</value>
 	</data>
 	<data name="MyGamesView__Ready_to_mod" xml:space="preserve">
-		<value>Gotowe do modowania</value>
+		<value>Dodane gry</value>
```

Comment:
This says 'added games'. Maybe it's a mistake.

Reply:
[#1140](https://github.com/Nexus-Mods/NexusMods.App/pull/1140)